### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by waldyrious**
- Center title to match the logo
- Change logo image to the PNG version, as SVG seems to be unsupported by PyPI:
  https://pypi.org/project/mwclient/
- Remove Landscape.io badge, as the service doesn't seem to be working:
  - https://landscape.io/github/mwclient/mwclient
  - https://github.com/landscapeio/landscape-issues/issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-<div align="center"><img src="docs/source/logo.svg" width="300"/></div>
-
-# mwclient
+<div align="center">
+    <img src="docs/source/logo.png"/>
```
Comment (danmichaelo) [inline README.md]: While we're at it, should we also add an `alt` attribute here?

**Comment by danmichaelo** [comment]
Thanks, yes, the Landscape service has always been quite unstable. It's a nice service when it works though, so I hope it will eventually recover and made a small donation to support the development, but I agree we can remove the the badges now, and rather add them back later if the service recovers.

File: README.md (hunk 1 above)
Comment (waldyrious) [reply README.md]: Good call, will do.

Comment (waldyrious) [reply README.md]: Done.

**Pull request merged**